ext/mysqli/mysqli_nonapi.c

```diff
@@ -1176,7 +1176,7 @@ PHP_FUNCTION(mysqli_begin_transaction)
 	size_t			name_len = -1;
 	zend_bool	err = FALSE;
 
-	if (zend_parse_method_parameters(ZEND_NUM_ARGS(), getThis(), "O|ls", &mysql_link, mysqli_link_class_entry, &flags, &name, &name_len) == FAILURE) {
+	if (zend_parse_method_parameters(ZEND_NUM_ARGS(), getThis(), "O|ls!", &mysql_link, mysqli_link_class_entry, &flags, &name, &name_len) == FAILURE) {
```

Comment:
Or alternatively, should we make the default `""`, and get rid of the `Savepoint name cannot be empty` warning (and pass `NULL` to `*_begin_transaction*()` in this case)?